doc/pipeline.rst

```diff
@@ -24,7 +24,12 @@ Input file format
 
 The input files containing feature values and scores for all responses in training and evaluation data should be in tabular format with features and scores stored in columns and each row correponding to a single response.
 
-RSMTool supports input files in ``.csv``, ``.tsv``, ``.sas7bdat``, or ``xls``/``.xlsx`` format. For Excel spreadsheets all data must be stored in the first sheet. The format of the file is determined based on the extension. In all cases the :ref:`output files<intermediate_files_rsmtool>` will be saved in ``.csv`` format by default (see :ref:`file format <file_format>` for other intermediate file output options).
+RSMTool supports input files in ``.csv``, ``.tsv``, ``.sas7bdat``, ``xls``/``.xlsx``, or ``.jsonlines`` format. The format of the file is determined based on the extension. In all cases the :ref:`output files<intermediate_files_rsmtool>` will be saved in ``.csv`` format by default (see :ref:`file format <file_format>` for other intermediate file output options).
```

Comment:
Ignore this change. It's an erroneous commit from another branch that is in PR #271 . At this point it will be faster to just move on to that commit.